kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeJobStatus.js

```diff
@@ -377,7 +380,7 @@ define([
             let ret = [];
             switch (type) {
                 case '[object String]':
-                    if (obj.match(/^[^\/]+\/[^\/]+(\/[^\/]+)?$/)) {
+                    if (obj.match(/^[^/]+\/[^/]+(\/[^/]+)?$/)) {
```

Comment:
unnecessary `\` in the `[ ]`

Reply:
Eventually we should make the object reference matchers just a utility function. I know there's an UPA matcher in the UPA api module, but it won't match workspace name / object name (or any name / id combination, which is also valid).

Anyway, this is fine. Just want to put that out there.